pkg/statusreconciler/controller.go

```diff
@@ -245,7 +245,7 @@ func (c *Controller) triggerNewPresubmits(addedPresubmits map[string][]config.Pr
 			// added. we know all of the presubmits we are filtering need to be forced to run, so we can
 			// enforce that with a custom filter
 			filter := pjutil.NewArbitraryFilter(func(p config.Presubmit) (shouldRun bool, forcedToRun bool, defaultBehavior bool) {
```

Comment:
can we update the comment also, like this:
```go
			// we want to appropriately trigger and skip from the set of identified presubmits that were
			// added. if a run_if_changed job has its regex pattern changed we should check PRs against the new regex pattern and
			// trigger if needed instead of forcing the job to run on all PRs. so the default behavior of the filter need to be false.
```